client/landing/site.landing/coffee/core/utils.coffee

```diff
@@ -425,4 +418,18 @@ module.exports = utils = {
 
     input.setWidth width
 
+
+  loadRecaptchaScript: (callback) ->
+
+    return callback()  if grecaptcha?
+
+    recaptchaLoadCallbacks.push callback
+    recaptchaScript = new kd.CustomHTMLView
+      tagName    : 'script'
+      attributes :
+        src      : RECAPTCHA_JS
+        async    : yes
+        defer    : yes
+
+    recaptchaScript.appendToDomBody()
```

Comment:
nope, it still causes multiple `script` tags. to prevent this on line 426 needs to be changed as follow;

```coffee
recaptchaLoadCallbacks.push callback
return  if recaptchaLoadCallbacks.length > 1
```